backend/uclapi/oauth/views.py

```diff
@@ -507,7 +506,9 @@ def userdata(request, *args, **kwargs):
         "upi": token.user.employee_id,
         "scope_number": token.scope.scope_number,
         "is_student": is_student,
-        "ucl_groups": token.user.raw_intranet_groups.split(';')
+        "ucl_groups": token.user.raw_intranet_groups.split(';'),
+        "sn": token.user.sn,
```

Comment:
`sn` is confusing as this might be confused for student number (rather than the intended **s**ur**n**ame or maybe **s**econd**n**ame?).

Reply:
Agreed, but I don't know if it stands for surname or second name (there's a difference I believe). Either way, they can read the docs I guess.